src/Context/Node/NodeContext.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * @author Sergii Bondarenko, <sb@firstvector.org>
+ */
+namespace Drupal\TqExtension\Context\Node;
+
+use Drupal\TqExtension\Context\RawTqContext;
+
+class NodeContext extends RawTqContext
+{
+    /**
+     * @When /^I (visit|view|edit) "([^"]+)" node of type "([^"]+)"$/
+     */
+    public function iVisitNodePageOfType($operation, $title, $type) {
+        if ($operation == 'visit') {
+            $operation = 'view';
+        }
+        $query = new \EntityFieldQuery();
+        $result = $query
+            ->entityCondition('entity_type', 'node')
+            // @todo: Add support for CT label.
+            ->entityCondition('bundle', strtolower($type))
+            ->propertyCondition('title', $title)
+            ->range(0, 1)
+            ->execute();
+
+        if (empty($result['node'])) {
+            $params = array(
+                '@title' => $title,
+                '@type' => $type,
+            );
+            throw new \Exception(format_string("Node @title of @type not found.", $params));
+        }
+
+        $nid = key($result['node']);
+        $this->getSession()->visit($this->locatePath('/node/' . $nid . '/' . $operation));
+    }
+
+    /**
+     * @When I edit this node
+     */
+    public function iEditThisNode()
```

Comment:
Do something like this, based on [this commit](https://github.com/BR0kEN-/TqExtension/commit/cbd66f7577d7cf7970af658ad0be47f5e0c93397).
```php
        $args = arg();

        if (count($args) < 2 || ('node' !== $args[0] && $args[1] <= 0)) {
            throw new \RuntimeException(sprintf('Page "%s" is not a node. Unable to edit.', static::$pageUrl));
        }

        $this->getRedirectContext()->visitPage('node/' . $args[1] . '/edit');
```